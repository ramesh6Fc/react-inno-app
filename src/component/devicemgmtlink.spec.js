import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';
import Devsimmgmt from './devicemgmtlink';


describe('The Device mnagement Link Component', () => {
    it ('Device mnagement Link component page renderer as expected',() => {
        const tree = renderer
                        .create(<Devsimmgmt />)
                        .toJSON();
                        console.log(tree.children, "Device mnagement Link component page elements");

                        expect(tree).toMatchSnapshot ();
    })

})