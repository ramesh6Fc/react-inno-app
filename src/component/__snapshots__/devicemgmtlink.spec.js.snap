// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Device mnagement Link Component Device mnagement Link component page renderer as expected 1`] = `
<div
  className="container-fluid"
>
  <div
    className="row"
  >
    <div
      className="col-lg-1"
    >
       
    </div>
    <div
      className="col-lg-11"
    >
      <ul
        className="d-flex justify-content-around align-right"
        style={
          Object {
            "list-style": "none",
          }
        }
      >
        <li
          className="takeaction social-link"
        >
          <div
            className="dropdown"
            style={
              Object {
                "fontSize": "13px",
              }
            }
          >
            <button
              aria-expanded={false}
              aria-haspopup={true}
              className="dropdown-toggle btn btn-primary"
              disabled={false}
              id="dropdown-basic-button"
              onClick={[Function]}
              type="button"
            >
              DEVICE ACTIONS
            </button>
          </div>
        </li>
        <li
          className="takeaction social-link"
        >
          <div
            className="dropdown"
            style={
              Object {
                "fontSize": "13px",
              }
            }
          >
            <button
              aria-expanded={false}
              aria-haspopup={true}
              className="dropdown-toggle btn btn-primary"
              disabled={false}
              id="dropdown-sim-action"
              onClick={[Function]}
              type="button"
            >
              SIM ACTIONS
            </button>
          </div>
        </li>
        <li />
      </ul>
    </div>
  </div>
</div>
`;
